forem/forem
# [15 minute fix] Now mention auto-complete dynamic imports are cached.

## What type of PR is this? (check all applicable)


## Description

A small performance improvement to avoid loading dynamic imports multiple times for the mention auto-complete feature as per this comment https://github.com/forem/forem/pull/13126/files#r601832869

## Related Tickets & Documents

#13126

## QA Instructions, Screenshots, Recordings

In any comment textarea in the application (posts, notifications), the mention auto-complete functionality should still work.

See the QA instructions from #13061

Other things to test:

1. Create a comment on an article, navigate the site a bit, come to another article or the same one again via navigation in the site and ensure that the mention auto-complete still works in the comments.

### UI accessibility concerns?

N/A

## Added tests?

- [x] No, and this is why: This is a refactor already covered by tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
- [x] This change does not need to be communicated, and this is why not: It's a small refactor that does not change the feature.

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![Johnny Rose from Shitt's Creek saying "Here we go! In the sky"](https://media.giphy.com/media/xUA7b9t2sRUhpFtjgc/giphy.gif)
